### PR TITLE
python: more small linting cleanups

### PR DESCRIPTION
**Pull request by allisonkarlitskaya**

### CODE REVIEW

File: test/pytest/test_code.py [hunk 1]
```diff
@@ -15,26 +15,22 @@
...
 
 
-@pytest.mark.skipif(not shutil.which('mypy'), reason='mypy is not installed')
-def test_bridge_mypy():
+@pytest.mark.parametrize('command', [
     # only test src/cockpit; src/systemd_ctypes does not have type annotations yet
     # disable caching, as it otherwise often crashes with "Cannot find component 'inotify' for 'systemd_ctypes...."
-    subprocess.check_call(['mypy', '--no-incremental', 'src/cockpit'], cwd=ROOT_DIR)
-
+    ['mypy', '--no-incremental', 'src/cockpit'],
```
Comment (martinpitt) [inline test/pytest/test_code.py:27]: Unnecessary newline

File: test/pytest/test_code.py [hunk 2]
```diff
@@ -26,6 +26,8 @@
     ['mypy', '--no-incremental', 'src/cockpit'],
 
     ['ruff', 'check', '--no-cache', '.'],
+
```
Comment (martinpitt) [inline test/pytest/test_code.py:29]: Another unnecessary newline, this makes the declaration appear oddly disconnected.

**Review by martinpitt — APPROVED** [review]
Thanks! If you need/want to push again, please fix the newlines, otherwise this is nice!

File: test/pytest/test_code.py (hunk 1 above)
Comment (allisonkarlitskaya) [reply test/pytest/test_code.py:27]: I did this because otherwise it was difficult to understand that comment above applied only to the one line.

**Pull request merged**